docker-compose.yml

```diff
@@ -13,6 +13,8 @@ crossdock:
 
       - BEHAVIOR_DANCE=client,server,transport
       - BEHAVIOR_SING=client,server,transport
+      - BEHAVIOR_RUN=client,server,transport
+      - BEHAVIOR_SKIP_RUN=client=alpha+server=alpha
```

Comment:
Yes, so : is used for assignment and + is used to separated multiple axis conditions in a filter.
Multiple filters are separated by a comma.